packages/cli/src/util/list/state-string.ts

```diff
@@ -0,0 +1,24 @@
+import chalk from 'chalk';
+import title from 'title';
+
+export function stateString(s: string) {
+  const CIRCLE = '● ';
+  const sTitle = s && title(s);
```

Comment:
`s` can't be undefined or null here (by type declaration) so remove the `s &&` part. 

Also, fwiw this kinda thing is a code smell IMO. `&&` should really only be used for conditional expressions. I'd rather see an explicit `s ? title(s) : 's is null/undefined case'` expression instead

Reply:
👍 Decided to remove the state string altogether after seeing Nate's comment